event-store/build.gradle.kts

```diff
@@ -16,9 +16,9 @@ tasks.test {
 
 dependencies {
   implementation("com.typesafe:config:1.3.2")
-  implementation(platform("org.hypertrace.core.kafkastreams.framework:kafka-bom:0.2.10"))
+  implementation(platform("org.hypertrace.core.kafkastreams.framework:kafka-bom:0.4.7"))
```

Comment:
Did we intentionally choose an old version of this? This version of the BOM includes older, vulnerable deps.